components/sendinblue/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@pipedream/sendinblue",
-  "version": "0.0.1",
+  "version": "0.0.3",
```

Comment:
Any reason this is being bumped to .3 instead of .2?

Reply:
Hi @GTFalcao, there is no reason for `0.3` I made it `0.2`.